app/views/catalog/_slideshow.html.erb

```diff
@@ -6,10 +6,10 @@
 
     <!-- Controls -->
     <a class="left carousel-control prev" href="#slideshow" data-slide="prev">
-      <%= blacklight_icon 'chevron_left' %>
+      <%= blacklight_icon 'chevron_left', { additional_options: { label_context: 'previous_image' }} %>
```

Comment:
@cbeer @jcoyne slight change here for backwards compatibility and hopefully future proofing

Reply:
😢 I'm not sure how to make this go green without a Blacklight release. Any suggestions?

Reply:
Not w/o adding some sort of wrapper that checked the `blacklight_icon` method's `parameters` and then adding the `additional_options` conditionally.  Not sure if that's ultimately worth it 🤷‍♂️

Reply:
Yeah.. doesn't seem like it.